plot_krun_results.py

```diff
@@ -77,9 +79,9 @@
 DPI = 300
 
 
-def main(is_interactive, data_dcts, percentiles, window_size,
-         level, num_samples, outfile, xlimits, single_exec,
-         with_outliers, benchmark=None):
+def main(is_interactive, data_dcts, window_size, outfile, xlimits, single_exec,
+         with_outliers, unique_outliers, mean=False, sigma=False,
+         benchmark=None):
```

Comment:
On a related note, #81 describes a refactoring and an improvement which could be made to main() here.